springwolf-plugins/springwolf-kafka-plugin/src/main/java/io/github/stavshamir/springwolf/asyncapi/types/channel/operation/message/header/AsyncHeadersForSpringKafkaBuilder.java

```diff
@@ -1,6 +1,6 @@
 package io.github.stavshamir.springwolf.asyncapi.types.channel.operation.message.header;
 
-import org.springframework.kafka.support.converter.AbstractJavaTypeMapper;
+import org.springframework.kafka.support.mapping.AbstractJavaTypeMapper;
```

Comment:
Due to the spring update, also spring-kafka is updated, which includes this change.

If we are assuming that springwolf users are up-to-date, then this change is good. Otherwise, this might force users to upgrade spring-kafka.

I am not sure if we want to include in the next release as it contains already so many changes: async annotations + spring properties + spring-cloud + fixes.

Or, this could be a minor update right after the upcoming release (1.1.0)